### PR TITLE
cleanup code

### PR DESCRIPTION
**Pull request by mikememo**

### CODE REVIEW

**Review by mikememo — COMMENTED** [review]
good job

File: WindowsFormsApp1/Form1.cs [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Forms;
-using WMPLib;
-using System.IO;
-using System.Globalization;
-
-namespace WindowsFormsApp1
+﻿namespace WindowsFormsApp1
 {
```
Comment (mikememo) [inline WindowsFormsApp1/Form1.cs:2]: i moved these inside the namespace

Comment (twilkus) [reply WindowsFormsApp1/Form1.cs:2]: k.

**Review by twilkus — APPROVED** [review]
This looks great.

**Pull request merged**